### ISSUE TITLE
Reproducibility issue: ADE/FDE higher than reported results on ETH/UCY

### ISSUE THREAD

**Issue by bool-luo** [Chenwangxing/PCHGCN-Master#3]
Hi, thanks for sharing your excellent work and code for 
"PCHGCN: Physically Constrained Higher-Order Graph Convolutional Network for Pedestrian Trajectory Prediction".

I am trying to reproduce the reported results, but I observed a noticeable performance gap.

## My Results
- ADE: 0.28
- FDE: 0.47

## Reported Results (Paper)
- ADE: 0.22
- FDE: 0.37

## Environment
- Dataset: ETH/UCY (hotel, univ, etc.)
- Code version: [latest GitHub version / commit id]
- GPU: [e.g., RTX 3090]
- PyTorch version: 2.51
- CUDA version: 11.8

## Training Details
- Command:  
  ```bash
  python train.py --use_lrschd

**Comment by Chenwangxing** [comment]
Can the weights provided in the code achieve the effects described in the paper? Nine months ago, I tried PCHGCN, which was able to achieve the effects described in the paper through normal training. Due to the long time that has passed, I will re-examine the code next week and provide an answer to help you solve your problem.

**Comment by Chenwangxing** [comment]
I just checked the code and retested it, and found no problems! The test code outputs results from the ETH, UCY, and SDD datasets by default. The average error output only displays correctly when the SDD result ('PCHGCN/sdd',) is masked. 
If you want the prediction results for ETH and UCY, you should mask the SDD result ('PCHGCN/sdd',). 
If you want the prediction results for SDD, you should mask the ETH and UCY result ('PCHGCN/eth', 'PCHGCN/hotel', 'PCHGCN/univ', 'PCHGCN/zara1', 'PCHGCN/zara2',).

**Comment by bool-luo** [comment]
Hi, thanks for your help!
I followed your suggestion and masked the SDD dataset during evaluation. After that, I was able to reproduce results much closer to those reported in the paper.
Really appreciate your clarification — it helped me identify the issue quickly. This will also be helpful for others trying to reproduce the results.
Thanks again for maintaining the code!
